backend/coins/btc/headers/headers.go

```diff
@@ -433,6 +436,7 @@ func (headers *Headers) VerifiedHeaderByHeight(height int) (*wire.BlockHeader, e
 }
 
 func (headers *Headers) kick() {
+	time.Sleep(30 * time.Second)
```

Comment:
Was this left here intentionally? In my test, it seems to slow down the sync, which takes forever.

Reply:
omg no, thanks for finding it. quite embarrassing :sweat_smile: